app/assets/javascripts/modules/gtm-track-copy.js

```diff
@@ -0,0 +1,25 @@
+function GTMCopyListener () { }
+
+GTMCopyListener.prototype.handleCopyEvent = function () {
+  window.dataLayer.push({
+    'event': 'text-copied'
```

Comment:
From my conversation with Andy he was suggesting that you could only append data to the dataLayer variable prior to GTM initialisation - which would suggest these method calls wouldn't have any effect aside from modifying our global variable.

Reply:
Events can be pushed at any time if a listener is setup in GTM to record them.

Reply:
I don't think that's the case, as we're pushing the value of the checked radio buttons on form submit and that's being tracked.

Reply:
Ah great - that's better than setting them in the header like we were doing.